Instruments/config/2017/Skimmer_tauPOG.cfg

```diff
@@ -134,7 +134,6 @@ file_ex: SUSYGluGluToHToTauTau-M-400_TuneCP5.root SUSYGluGluToHToTauTau_M-400_Tu
 file_ex: SUSYGluGluToHToTauTau-M-600_TuneCP5.root SUSYGluGluToHToTauTau_M-600_TuneCP5.root
 file_ex: SUSYGluGluToHToTauTau-M-700_TuneCP5.root SUSYGluGluToHToTauTau_M-700_TuneCP5.root
 file_ex: SUSYGluGluToHToTauTau-M-800_TuneCP5.root SUSYGluGluToHToTauTau_M-800_TuneCP5.root
-file_ex: SUSYGluGluToHToTauTau-M-900_TuneCP5.root SUSYGluGluToHToTauTau_M-900_TuneCP5.root
```

Comment:
if the sample is missing due to some error that is relevant only to the current production, comment it instead of removing.